### PR TITLE
small tweaks to get docs ready for an open beta

### PR DESCRIPTION
**Pull request by EmmaOnThursday**
### What does this PR do?
- remove note about asking CSM about feature
- make "Node.js" consistent across whole page
- move Node.js tab to end to match table
- update node tab to reference correct tracer version (3.13 not 3.9)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 1]
```diff
@@ -262,16 +262,51 @@ cursor.executemany("select %s", (("foo",), ("bar",)))
 
 {{% /tab %}}
 
+{{% tab ".NET" %}}
+
+<div class="alert alert-warning">
+This features requires automatic instrumentation to be enabled for your .NET service.
```
Comment (estherk15) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:268]:
```suggestion
This feature requires automatic instrumentation to be enabled for your .NET service.
```

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 2]
```diff
@@ -262,16 +262,51 @@ cursor.executemany("select %s", (("foo",), ("bar",)))
...
+</div>
+
+Follow the [.NET Framework tracing instructions][1] or the [.NET Core tracing instructions][2] to install the automatic instrumentation package and enable tracing for your service.
+
+Ensure that you are using a supported client library (e.g. `Npgsql`).
```
Comment (estherk15) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:273]:
```suggestion
Ensure that you are using a supported client library. For example, `Npgsql`.
```

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 3]
```diff
@@ -262,16 +262,51 @@ cursor.executemany("select %s", (("foo",), ("bar",)))
...
+
+[1]: /tracing/trace_collection/dd_libraries/dotnet-framework
+[2]: /tracing/trace_collection/dd_libraries/dotnet-core
+
+{{% /tab %}}
+
+{{% tab "PHP" %}}
+
+<div class="alert alert-warning">
+This features requires the tracer extension to be enabled for your PHP service.
```
Comment (estherk15) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:286]:
```suggestion
This feature requires the tracer extension to be enabled for your PHP service.
```

File: content/en/database_monitoring/guide/connect_dbm_and_apm.md [hunk 4]
```diff
@@ -262,16 +262,51 @@ cursor.executemany("select %s", (("foo",), ("bar",)))
...
+
 {{% tab "Node.js" %}}
 
 <div class="alert alert-warning">
-Node is in alpha release and may be unstable.
+DBM-APM linking for Node.js is in alpha release and may be unstable.
 </div>
 
-Install or udpate [dd-trace-js][1] to version greater than `3.9.0` (or `2.22.0` if using end-of-life Node.js version 12):
+Install or udpate [dd-trace-js][1] to version greater than `3.13.0` (or `2.22.0` if using end-of-life Node.js version 12):
```
Comment (estherk15) [inline content/en/database_monitoring/guide/connect_dbm_and_apm.md:306]:
```suggestion
Install or update [dd-trace-js][1] to a version greater than `3.13.0` (or `2.22.0` if using end-of-life Node.js version 12):
```

**Review by estherk15 — CHANGES_REQUESTED** [review]

**Comment by EmmaOnThursday** [comment]
@estherk15 I noticed this page is listed as `private`, but we've already added it to the docs sidebar under Guides. What does that mean in practice?

**Comment by estherk15** [comment]
@EmmaOnThursday It means it's not searchable. If you DON'T want this visible in the public docs we can remove it from sidebar. Or if you DO want this visible, you can remove the `private:true` from the frontmatter. Let me know if you have any questions on this!

**Review by estherk15 — APPROVED** [review]

**Review by alexandre-normand — APPROVED** [review]

**Comment by EmmaOnThursday** [comment]
@estherk15 if you are happy with these changes can you merge this?

**Pull request merged**